### ISSUE TITLE
Copy row with generated columns

### ISSUE THREAD

**Issue by fglueck** [dbeaver/dbeaver#34794]
### Description

If you copy a row from a table with a generated column, than this generated column would "saved/changed", too.

I mean, the generated column will be in the INSERT INTO statement, this throws an error.

### DBeaver Version

CE 24.1.2 and all before

### Operating System

any

### Database and driver

10.11.8-MariaDB-log

### Steps to reproduce

Sample: 

```
CREATE TABLE test1 (
  first varchar(20)
, last varchar(20)
, together varchar(40) GENERATED ALWAYS AS (CONCAT(first, ' ', last))
)

INSERT INTO cockpit.test1 (`first`,`last`)
	VALUES ('Frank','Glück');

SELECT *
FROM test1

-- copy row (Ctrl+Alt+Insert)
-- edit some
-- generates following SQL:
--  Auto-generated SQL script #202407121253
INSERT INTO cockpit.test1 (`first`,`last`,together)
	VALUES ('Frank','Pech','Frank Glück');

```

**Comment by uslss** [comment]
thanks for the report.
The issue is reproducible if changes are not saved.

https://github.com/user-attachments/assets/ad35fd48-2ad4-47dc-ad2c-53536475c6b7

**Comment by ixavierone** [comment]
yep, a really annoying thing for me, too. I have to always manually set the cell to [Null] to temporarily solve this problem. Of course, it can be avoided by using SQL code, but as a layman, I use Dbeaver coz I want to use a visual interface to operate.
